mne/gui/__init__.py

```diff
@@ -247,6 +247,48 @@ def locate_ieeg(info, trans, aligned_ct, subject=None, subjects_dir=None,
     return gui
 
 
+@verbose
+def view_stc(stc, subject=None, subjects_dir=None, src=None,
+             inst=None, show=True, block=False, verbose=None):
+    """Locate intracranial electrode contacts.
+
+    Parameters
+    ----------
+    stc : ndarray of shape (n_epochs, n_vertices, n_orient, n_freqs, n_times)
```

Comment:
`stc` in MNE always means some variant of `*SourceEstimate`, so I don't think this name is quite appropriate. Maybe `source_data` or something

Reply:
This is copying from `mne.viz.Brain.add_data`. The thought was that a time-frequency source estimate would be added in the future by @drammock and so I thought that this naming scheme would be forward compatible with that.

Reply:
>This is copying from mne.viz.Brain.add_data

The first arg there appears to be named `array` not `stc`

https://mne.tools/stable/generated/mne.viz.Brain.html#mne.viz.Brain.add_data

If you do `git grep "stc : "` you'll see what I mean
```
$ git grep "stc : "
examples/inverse/custom_inverse_solver.py:    stc : instance of SourceEstimate
examples/inverse/custom_inverse_solver.py:    stc : instance of SourceEstimate
mne/beamformer/_dics.py:    stc : SourceEstimate | VolSourceEstimate | list
mne/beamformer/_dics.py:    stc : SourceEstimate
mne/beamformer/_lcmv.py:    stc : SourceEstimate | VolSourceEstimate | VectorSourceEstimate
mne/beamformer/_lcmv.py:    stc : SourceEstimate | VolSourceEstimate
mne/beamformer/_lcmv.py:    stc : SourceEstimate | VolSourceEstimate
mne/forward/_make_forward.py:    stc : instance of VolSourceEstimate | list of VolSourceEstimate
mne/forward/forward.py:    stc : SourceEstimate
mne/forward/forward.py:    stc : SourceEstimate
mne/forward/forward.py:    stc : instance of SourceEstimate
...
```

Reply:
That's fine to name it `array`, I'll change it.

Reply:
I think this one is not quite resolved yet. The `stc->array` change helped, as did `inst` only accepting `*TFR` objects. But the name is still `view_stc`, which still seems problematic.